Framework/WorkflowAlgorithms/src/AlignAndFocusPowder.cpp

```diff
@@ -116,10 +123,30 @@ void AlignAndFocusPowder::init() {
                   "Width of events (in "
                   "microseconds) near the prompt "
                   "pulse to remove. 0 disables");
-  declareProperty("CompressTolerance", 0.01,
+  auto mustBePositive = boost::make_shared<BoundedValidator<double>>();
+  mustBePositive->setLower(0.0);
+  declareProperty(make_unique<PropertyWithValue<double>>("CompressTolerance",
+                                                         1e-5, mustBePositive,
+                                                         Direction::Input),
                   "Compress events (in "
                   "microseconds) within this "
-                  "tolerance. (Default 0.01) ");
+                  "tolerance. (Default 0.01)");
```

Comment:
Looks like default changed from 0.01 -> 1e-5. Shouldn't this line change this accordingly?